docker/run/Dockerfile

```diff
@@ -50,7 +51,9 @@ ENV \
     # Installation directories.
     FLUTTER_HOME=/opt/flutter
 
-ENV PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin"
+ENV \
```

Comment:
Nit: We usually don't break the line before the first statement, see e.g. the `RUN` statements.

Reply:
I thought I'd make it consistent with the ENV statement directly above. Have you noticed that one or rather still want to change it ?

Reply:
I indeed overlooked that; then keep it as-is.